blog/2017-08-10-RealTimePlanningUsingKieServerPart1.adoc

```diff
@@ -0,0 +1,149 @@
+= Optimize your problems using KIE Server, part 1
+:page-interpolate: true
+:awestruct-author: mcimbora
+:awestruct-layout: blogPostBase
+:awestruct-tags: [feature, howto, kie server]
+
+KIE Server provides an execution environment separated from your client application.
+Run it on the same local network as your client, or in the cloud, depending on your needs.
+The server capabilities are exposed via JMS/REST interface, with the latter making the content easily consumable
+by non-Java clients.
+This is extremely useful if you want to leverage OptaPlanner outside Java ecosystem.
+In the 7 release stream, we worked to improve OptaPlanner/KIE Server integration.
+Apart from API improvements I am happy to announce the server now supports real-time planning.
+Stay tuned for part two.
+
+== New OptaPlanner REST API overview
```

Comment:
This is a dry chapter to start with. It's like a dictionary. Note sure if we even need it in a blog post. We need it in the reference manual, as this is the menu from which to chose from. But a blog article should be a story, something interesting (even exciting) to read, it can leave out of details, it doesn't never to cover all options, it just needs to cover 1 story arc, 1 use case, 1 value proposition.